doc/port.1.txt

```diff
@@ -178,7 +178,7 @@ The port command recognizes several global flags and options.
 -t::
     Enable trace mode debug facilities on platforms that support it, currently
     only Mac OS X.
-    + 
+    +
```

Comment:
I think this is a change of behavior. While trailing whitespace is uncommon, it's the only way in asciidoc to add a newline to a paragraph without ending the paragraph.

Can you check the output to make sure that the text below this linebreak is still associated with the `-t` command line option?

Reply:
I checked the man page before and after removing the trailing whitespace and it looks same to me.

Screenshot of comparison: https://imgur.com/a/u671e

Reply:
OK, thanks for checking.